### PR TITLE
Order column selection dropdown alphabetically, add test

### PR DESCRIPTION
**Pull request by fergaldoyle**
- Order the items in column selection menu in the data grid alphabetically
- Added a test

### CODE REVIEW

File: app/view/grid/Grid.js [hunk 1]
```diff
@@ -236,5 +237,12 @@ Ext.define('CpsiMapview.view.grid.Grid', {
                 boxLabel: 'Page Records?',
                 handler: 'togglePaging'
             }]
-    }]
+    }],
+
+    constructor: function() {
```
Comment (geographika) [inline app/view/grid/Grid.js:242]: Out of interest why is this required? Shouldn't the default constructor be called anyway?

File: test/spec/controller/grid/Grid.spec.js [hunk 2]
```diff
@@ -9,4 +9,54 @@ describe('CpsiMapview.controller.grid.Grid', function() {
...
+
+            // get a reference to the menu by taking it from the spy arguments
+            var menu = onHeaderMenuCreateSpy.getCalls()[0].args[1];
+            var menuItems = menu.down('[itemId=columnItem]').menu.items.items;
+            var orderedTitles = menuItems.map(function (item) {
+                return item.text;
+            })
+
+            // deep array comparison - original order without sorting would be [ 'Test1', 'Test3', 'Test2' ]
+            expect(orderedTitles).to.eql(['Test1', 'Test2', 'Test3']);
```
Comment (geographika) [inline test/spec/controller/grid/Grid.spec.js:55]: Very nice!

File: app/view/grid/Grid.js (hunk 1 above)
Comment (fergaldoyle) [reply app/view/grid/Grid.js]: That's just a left over from debugging. I've removed it now in the latest commit.

**Review by geographika — APPROVED** [review]
